hddm/models/hddm_info.py

```diff
@@ -102,10 +102,10 @@ class HDDM(HDDMBase):
     def __init__(self, *args, **kwargs):
         self.slice_widths = {'a':1, 't':0.01, 'a_std': 1, 't_std': 0.15, 'sz': 1.1, 'v': 1.5,
                              'st': 0.1, 'sv': 3, 'z_trans': 0.2, 'z': 0.1,
-                             'p_outlier':1., 'v_std': 1,'alpha':3.5,'dual_alpha':3.5}
+                             'p_outlier':1., 'v_std': 1, 'alpha':3.5, 'dual_alpha':3.5}
         self.emcee_dispersions = {'a':1, 't': 0.1, 'a_std': 1, 't_std': 0.15, 'sz': 1.1, 'v': 1.5,
                                   'st': 0.1, 'sv': 3, 'z_trans': 0.2, 'z': 0.1,
-                                  'p_outlier':1., 'v_std': 1,'alpha':3.5,'dual_alpha':3.5}
+                                  'p_outlier':1., 'v_std': 1, 'alpha':3.5, 'dual_alpha':3.5}
```

Comment:
can remove `dual_alpha` here and above.

Reply:
Yes, or actually, I'll replace it with pos_alpha, which is the parameter being created if dual is set to True for HDDMrl and Hrl.